vulkaninfo/vulkaninfo.c

```diff
@@ -2564,6 +2567,116 @@ static void AppGpuDumpProps(const struct AppGpu *gpu, FILE *out) {
+                            (uint32_t)id_props->driverUUID[12],
+                            (uint32_t)id_props->driverUUID[13],
+                            (uint32_t)id_props->driverUUID[14],
+                            (uint32_t)id_props->driverUUID[15]);
+                    fprintf(out, "\t\t\t\t\t\t<details><summary>deviceLUIDValid = <div class='val'>%s</div></summary></details>\n",
+                            id_props->deviceLUIDValid ? "true" : "false" );
+                    if (id_props->deviceLUIDValid) {
+                        fprintf(out, "\t\t\t\t\t\t<details><summary>deviceLUID      = <div class='val'>%02x%02x%02x%02x-%02x%02x%02x%02x</div></summary></details>\n",
+                                (uint32_t)id_props->deviceLUID[0],
+                                (uint32_t)id_props->deviceLUID[1],
+                                (uint32_t)id_props->deviceLUID[2],
+                                (uint32_t)id_props->deviceLUID[3],
+                                (uint32_t)id_props->deviceLUID[4],
+                                (uint32_t)id_props->deviceLUID[5],
+                                (uint32_t)id_props->deviceLUID[6],
+                                (uint32_t)id_props->deviceLUID[7]);
+                        fprintf(out, "\t\t\t\t\t\t<details><summary>deviceNodeMask  = <div class='val'>0x%08x</div></summary></details>\n",
+                                id_props->deviceNodeMask);
+                    }
+                    fprintf(out, "\t\t\t\t\t</details>\n");
+                } else if (human_readable_output) {
+                    printf("\nVkPhysicalDeviceIDProperties:\n");
+                    printf("=========================================\n");
+                    printf("\tdeviceUUID      = %02x%02x%02x%02x-%02x%02x-%02x%02x-%02x%02x-%02x%02x%02x%02x%02x%02x\n",
+                            (uint32_t)id_props->deviceUUID[0],
+                            (uint32_t)id_props->deviceUUID[1],
+                            (uint32_t)id_props->deviceUUID[2],
+                            (uint32_t)id_props->deviceUUID[3],
+                            (uint32_t)id_props->deviceUUID[4],
+                            (uint32_t)id_props->deviceUUID[5],
+                            (uint32_t)id_props->deviceUUID[6],
+                            (uint32_t)id_props->deviceUUID[7],
+                            (uint32_t)id_props->deviceUUID[8],
+                            (uint32_t)id_props->deviceUUID[9],
+                            (uint32_t)id_props->deviceUUID[10],
+                            (uint32_t)id_props->deviceUUID[11],
+                            (uint32_t)id_props->deviceUUID[12],
+                            (uint32_t)id_props->deviceUUID[13],
+                            (uint32_t)id_props->deviceUUID[14],
+                            (uint32_t)id_props->deviceUUID[15]);
+                    printf("\tdriverUUID      = %02x%02x%02x%02x-%02x%02x-%02x%02x-%02x%02x-%02x%02x%02x%02x%02x%02x\n",
```

Comment:
For loop here.

Reply:
Just so I don't have to do this twice, are you suggesting something like:

```
for (i = 0 .. 15) {
   switch (i) {
   case 4:
   case 6:
   case 8:
   case 10:
      printf("-");
      break;
   default;
      break;
   }
   printf("%02x", (uint32_t)uuid[i]);
}
```
Or a bunch of small for loops for each section of the UUIDs?  Like the formatting changes I just fixed, neither of these really seem clearer/more readable to me, but I'll write it however you prefer.

Reply:
My gut reaction was that it would be more maintainable to use a for loop than go through an entire array item by item.  On a second look, these are IDs and probably not going to have any updates that require complex or major changes to the code in the future.  If it's passing the clang-format check now then it LGTM.